src/frontend/Deprecation_analysis.mli

```diff
@@ -0,0 +1,28 @@
+open Core_kernel
+open Ast
+
+val deprecated_userdefined : string String.Table.t
+val distribution_suffix : string -> bool
+val without_suffix : string sexp_list -> string -> string
+val is_distribution : string -> bool
+val rename_distribution : string -> string
+val rename_function : string -> string
+
+val userdef_distributions :
+     ( (located_meta, unit) expr_with
+     , located_meta
+     , ((located_meta, unit) expr_with, located_meta) lval_with
+     , unit )
+     statement_with
+     sexp_list
+     sexp_option
+  -> string sexp_list
```

Comment:
How did you get these types? Is there a way to generate an .mli file? I think merlin will sometimes show "sexp_list" etc but we don't need to write the `sexp_` prefix for these three.

Reply:
I've tried to automatically generate the mli file with `ocamlc -i`, but got `Error: Unbound module Core_kernel`, so I added it myself until the type checker was happy and things made sense. I'm sure there's a better way. :-)